src/restaurant_inspections_api/services.clj

```diff
@@ -84,11 +84,15 @@
   (db/select-counties-summary))
 
 (defn- format-businesses-params
-  [params-map]
-  (assoc params-map
-         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
-                     (str/split zipCodes #","))
-         :page (* (:page params-map) (:perPage params-map))))
+  [{zip-codes :zipCodes
+    per-page :perPage
+    page :page
+    county-number :countyNumber}]
+  (assoc {}
```

Comment:
You dont need `(assoc {}` here, I think you can just wrap the whole thing in {}:
```
{:perPage per-page
 :countyNumber ...
 }```